rdr_service/genomic/genomic_job_controller.py

```diff
@@ -1910,6 +1910,26 @@ def check_aw3_ready_missing_files(self):
             )
         self.job_result = GenomicSubProcessResult.SUCCESS
 
+    def check_appointments_gror_revoked(self):
+        """ Alerts Color when a participant with scheduled appointment has revoked GRoR consent """
+        appointments_dao = GenomicAppointmentEventDao()
+        notification_email_address = config.getSetting(config.GENOMIC_APPOINTMENT_GROR_EMAIL, default=None)
+        revoked_appointments = appointments_dao.get_appointments_gror_revoked()
+        if notification_email_address and revoked_appointments:
```

Comment:
We need to return here if this condition is False the subprocess of `GenomicSubProcessResult.NO_RESULTS` for the job result.